packages/@aws-cdk/core/lib/private/runtime-info.ts

```diff
@@ -38,6 +39,33 @@ export function constructInfoFromConstruct(construct: IConstruct): ConstructInfo
   return undefined;
 }
 
+/**
+ * Add analytics data for any validation plugins that are used.
+ * Since validation plugins are not constructs we have to handle them
+ * as a special case
+ */
+function addValidationPluginInfo(stack: Stack, allConstructInfos: ConstructInfo[]): void {
+  let stage = Stage.of(stack);
+  let done = false;
+  do {
+    if (App.isApp(stage)) {
+      done = true;
+    }
+    if (stage) {
+      allConstructInfos.push(...stage.validationPlugins.map(
+        plugin => {
+          return {
+            fqn: `validation.${plugin.name}`,
+            // TODO: what version should we have here?
+            version: '0.0.0',
```

Comment:
Not sure what version makes sense here, maybe whatever version the framework is?

Reply:
What if we add a `version` property to `IValidationPlugin`?

Reply:
I thought about that, but then the plugin would have to somehow update that property when a new version is published.

Reply:
You're right. In that case, we should probably just leave `0.0.0` as a placeholder. Using the framework version would be confusing, and we already have that information elsewhere.

Reply:
We just don't want to mess up our analytics service which expects a certain format for this field.

Reply:
Ok last comment I make about this I promise :)

What if we have an optional `version` property on the plugin interface? For those plugin authors that want implement it, great, and for those that don't, we will use `0.0.0` and mention that this will mean they won't have plugin version information in the report. 

In the plugins we vend in TypeScript, we can easily implement it, so it would provide benefit.